src/fides/ctl/core/config/__init__.py

```diff
@@ -46,7 +43,9 @@ class FidesConfig(BaseModel):
     logging: LoggingSettings = LoggingSettings()
     notifications: NotificationSettings = NotificationSettings()
     redis: RedisSettings = RedisSettings()
-    security: SecuritySettings = {}  # type: ignore
+    # Use 'construct' to avoid running validation
+    # This allows us to use the correct type but populate later
+    security: SecuritySettings = SecuritySettings.construct()
```

Comment:
@sanders41 i think this was the final solution...doing this here and then making sure to populate the settings correctly below